tools/mason/MasonPublish.chpl

```diff
@@ -267,6 +267,15 @@ proc dryRun(username: string, registryPath : string, isLocal : bool) throws {
     }
   }
   else {
+    const spacer = '------------------------------------------------------';
+    writeln('Checking Registry with ' + registryPath + ' path.');
+    var registryTest = registryPathCheck(registryPath, username, false);
+    writeln(spacer);
+    writeln('The current mason environment is:');
+    returnMasonEnv();
+    if MASON_REGISTRY.size == 1 {
```

Comment:
Should this be checking if `MASON_REGISTRY['mason-registry'] == https://github.com/chapel-lang/mason-registry` ? I am thinking a user could set `MASON_REGISTRY` to their local registry and this check would evaluate to true, yet the suggestion below would no longer be valid.